etc/config/compiler-explorer.defaults.properties

```diff
@@ -18,6 +18,7 @@ urlShortenService=default
 storageSolution=local
 localStorageFolder=./lib/storage/data/
 showSponsors=false
+compilerCacheConfig=OnDisk(out/compiler-cache,1024)
```

Comment:
Nice - I was doing this locally from the day I implemented the compiler cache, not sure why I never thought to make it the default...